src/main/java/com/sap/oss/phosphor/fosstars/data/github/VulnerabilitiesFromOwasp.java

```diff
@@ -0,0 +1,279 @@
+package com.sap.oss.phosphor.fosstars.data.github;
+
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.VULNERABILITIES;
+import static com.sap.oss.phosphor.fosstars.model.other.Utils.setOf;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sap.oss.phosphor.fosstars.data.AbstractArtifactInfoProvider;
+import com.sap.oss.phosphor.fosstars.data.DataProvider;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.Dependency;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.OwaspEntry;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.OwaspReference;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.OwaspVulnerability;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.Software;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.VulnerableSoftware;
+import com.sap.oss.phosphor.fosstars.model.Feature;
+import com.sap.oss.phosphor.fosstars.model.ValueSet;
+import com.sap.oss.phosphor.fosstars.model.subject.oss.MavenArtifact;
+import com.sap.oss.phosphor.fosstars.model.value.CVSS;
+import com.sap.oss.phosphor.fosstars.model.value.Reference;
+import com.sap.oss.phosphor.fosstars.model.value.ValueHashSet;
+import com.sap.oss.phosphor.fosstars.model.value.VersionRange;
+import com.sap.oss.phosphor.fosstars.model.value.Vulnerabilities;
+import com.sap.oss.phosphor.fosstars.model.value.Vulnerability;
+import com.sap.oss.phosphor.fosstars.model.value.Vulnerability.Builder;
+import com.sap.oss.phosphor.fosstars.model.value.Vulnerability.Resolution;
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.owasp.dependencycheck.Engine;
+import org.owasp.dependencycheck.exception.ExceptionCollection;
+import org.owasp.dependencycheck.exception.ReportException;
+import org.owasp.dependencycheck.utils.Settings;
+
+/**
+ * This data provider tries to fill out the
+ * {@link com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures#VULNERABILITIES}. This data
+ * provider gathers vulnerabilities about {@link MavenArtifact} using OWASP dependency.
+ */
+public class VulnerabilitiesFromOwasp extends AbstractArtifactInfoProvider<MavenArtifact> {
+
+  /**
+   * The base directory to contain the OWASP database and NVD data.
+   */
+  private static final String BASE_REPORT_DIR = ".fosstars";
```

Comment:
OWASP doesn't have a database. Looks like `BASE_REPORT_DIR` is an output directory for OWASP Dependency Check. Could you please update the javaodc?

Also, `.fosstars` is used by the command-line tool. Please use a subdirectory.

Reply:
I changed the name of the variable. 
- I confirmed that if NVD data source files is already downloaded from `NVD`, then OWASP Dependency-Check will use the same downloaded files to build it own index DB.